docs/source/tutorial/basic.rst

```diff
@@ -2,21 +2,586 @@
+
+.. code-block:: bash
+
+      PK  Created    Process label             Process State    Process status
+    ----  ---------  ------------------------  ---------------  ----------------
+       3  6m ago     multiply                  ⏹ Finished [0]
+       7  2m ago     ArithmeticAddCalculation  ⏹ Finished [0]
+      12  1m ago     ArithmeticAddCalculation  ⏹ Finished [0]
+
+    Total results: 3
+
+    Info: last time an entry changed state: 14s ago (at 09:07:45 on 2020-05-13)
+
+Workflows
+=========
+
+So far we have executed each process manually.
+AiiDA allows us to automate these steps by linking them together in a *workflow*, whose provenance is stored to ensure reproducibility.
+For this tutorial we have prepared a basic ``WorkChain`` that is already implemented in ``aiida-core``.
+You can see the code below:
+
+.. accordion:: MultiplyAddWorkChain code
+
+    .. code-block:: python
+
+        from aiida.orm import Code, Int
+        from aiida.engine import calcfunction, WorkChain, ToContext
+
+        @calcfunction
+        def multiply(x, y):
+            return x * y
+
+        class MultiplyAddWorkChain(WorkChain):
+            """WorkChain to perform basic arithmetic for testing and demonstration purposes."""
+
+            @classmethod
+            def define(cls, spec):
+                """Specify inputs and outputs."""
+                super(MultiplyAddWorkChain, cls).define(spec)
+                spec.input('x', valid_type=Int)
+                spec.input('y', valid_type=Int)
+                spec.input('z', valid_type=Int)
+                spec.input('code', valid_type=Code)
+                spec.outline(cls.multiply, cls.add, cls.result)
+                spec.output('result', valid_type=Int)
+
+            def multiply(self):
+                """Multiply two integers."""
+                self.ctx.multiple = multiply(self.inputs.x, self.inputs.y)
+
+            def add(self):
+                """Add two numbers with the ArithmeticAddCalculation process."""
+
+                builder = self.inputs.code.get_builder()
+
+                builder.x = self.ctx.multiple
+                builder.y = self.inputs.z
+
+                future = self.submit(builder)
+
+                return ToContext({'addition': future})
+
+            def result(self):
+                self.out('result', self.ctx['addition'].get_outgoing().get_node_by_label('sum'))
+
+    First, we recognize the ``multiply`` function we have used earlier, decorated as a ``calcfunction``.
+    The ``define`` class method specifies the ``input`` and ``output`` of the ``WorkChain``, as well as the ``outline``, which are the steps of the workflow.
+    These steps are provided as methods of the ``MultiplyAddWorkChain`` class.
+
+.. note::
+
+    Besides WorkChain's, workflows can also be implemented as *work functions*.
+    These are ideal for workflows that are not very computationally intensive and can be easily implemented in a Python function.
```

Comment:
I think we need to add some space after the dropdown:
![Screen Shot 2020-05-25 at 2 35 48 PM 1](https://user-images.githubusercontent.com/1441208/82813332-1fdb4180-9e95-11ea-87f4-2ae261235cbd.png)